users/views.py

```diff
@@ -81,4 +83,79 @@ def post(self, request):
             return JsonResponse({'message':'KEY_ERROR'}, status=400)
         
         except User.DoesNotExist:
-            return JsonResponse({'message':'INVALID_USER'}, status=401)
+            return JsonResponse({'message':'INVALID_USER'}, status = 401)
+
+
+class CartView(View):    
+    @signin_decorator
+    def post(self, request):
+        
+        try:
+            data         = json.loads(request.body)
+            product_name = data['product_name']
+            sizeup       = data['sizeup']
+            product = Product.objects.get(relative_product__name = product_name)
+
+            if sizeup:
+                product = Product.objects.get(relative_product__name = product_name)
+            
+                if not product:
+                    return JsonResponse({'message': 'SIZEUP_INVALID'}, status = 401)
+
+            cart, created = Cart.objects.get_or_create(user = request.user, product_id = product.id, defaults={'quantity': 1})
+            
+            if not created:
+                cart_quantity = cart.quantity
+                Cart.objects.filter(user = request.user, product_id = product.id).update(quantity = cart_quantity+1)
+                return JsonResponse({"message" : "ALREADY_EXIST"}, status = 200)
```

Comment:
굳이 이부분에 메세지를 담아주고 싶다면
`이미 존재합니다!` 라는 실패의 느낌을 주는 `ALREADY_EXISTS` 에러 메세지보다는 `이미 존재하기 때문에 그걸 업데이트했다!` 라는 의미의 메세지가 더 어울릴 것 같습니다. 
```suggestion
                return JsonResponse({"message" : "UPDATED"}, status = 200)
```